packages/Atoms/Global/README.md

```diff
@@ -20,6 +20,7 @@ npm install @dpc-sdp/ripple-global --save
 ├── masonry-layout
 ├── moment
 ├── object-fit-images
+├── vue-carousel
```

Comment:
Hey @alan-cole I think we should leave vue-carousel in card package and gallery package. Not need to be in the core.